app/modules/new-vm/index.coffee

```diff
@@ -270,6 +282,9 @@ module.exports = angular.module 'xoWebApp.newVm', [
           # FIXME: handles invalid entries.
           data.memory = memory
 
+        if $scope.cloudConfig
+          xo.vm.createCloudConfigDrive id, $scope.firstSR, $scope.cloudConfig
+
```

Comment:
@julien-f I can't remember when to use `$scope` or not. Please review and fix using the best practice. Thanks!